src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -1129,6 +1129,30 @@ public static class Biomes {
         public static final TagKey<Biome> IS_DRY_NETHER = tag("is_dry/nether");
         public static final TagKey<Biome> IS_DRY_END = tag("is_dry/end");
 
+        /**
+         * Biomes that are primarily composed of a specific wood type.
+         * For example, normal Forest biomes are mostly Oak Trees with a few Birch Trees. This biome would be in `c:primary_wood_type/oak` tag due to Oak dominance.
+         * For biomes that are composed of multiple wood types but in equal proportions, put the biome into multiple wood type tags. Biomes with very few trees like Plains are skipped.
+         *
+         * <p></p>
```

Comment:
You should really stop doing this. :smile:

Reply:
I keep forgetting